src/dmd/hdrgen.d

```diff
@@ -3433,3 +3438,24 @@ extern (C++) const(char)* parametersTypeToChars(Parameters* parameters, int vara
     v.parametersToBuffer(parameters, varargs);
     return buf.extractString();
 }
+
+/** Pretty print function parameters.
+ * Params:
+ *  parameter = parameter to print.
+ *  varargs = kind of varargs, see TypeFunction.varargs.
+ *  fullQual = whether to fully qualify types.
+ * Returns: Null-terminated string representing parameters. */
+extern (C++) const(char)* parameterToChars(Parameter parameter, int varargs, bool fullQual)
+{
+    OutBuffer buf;
+    HdrGenState hgs;
+    hgs.fullQual = fullQual;
+    scope PrettyPrintVisitor v = new PrettyPrintVisitor(&buf, &hgs);
+
+    parameter.accept(v);
+    if (varargs)
+    {
+        buf.writestring("...");
```

Comment:
need test case to cover this new line

Reply:
Added test, turns out it was wrong for variadic parameters :-/